cameo/flux_analysis/simulation.py

```diff
@@ -178,8 +178,9 @@ def create_constraint(model, constraint_id, var, reaction, flux_value):
                 return constraint
 
             def update_constraint(model, constraint, var, reaction, flux_value):
-                constraint.lb = flux_value
-                constraint.ub = flux_value
+                if constraint.lb != flux_value:
+                    constraint.lb = flux_value
+                    constraint.ub = flux_value
```

Comment:
No. It has to be always both. If one is, the other is too (look at the create function). There is no need to test both.